esrally/mechanic/telemetry.py

```diff
@@ -481,107 +481,90 @@ def record(self):
         current_sample = self.sample()
         for node_stats in current_sample:
             node_name = node_stats["name"]
+            collected_node_stats = {}
+
             if self.include_indices:
-                self.record_indices_stats(node_name, node_stats,
-                                          include=["docs", "store", "indexing", "search", "merges", "query_cache", "fielddata",
-                                                   "segments", "translog", "request_cache"])
+                collected_node_stats.update(
+                    self.collect_indices_stats(node_name, node_stats,
```

Comment:
Maybe we could benefit from shorter names by removing the `collect_` prefix from all methods? To some extent these methods are more like extractors or getters so I think it would be ok here.